documentation/docs/src/clients/socks5-client.md

```diff
@@ -52,13 +52,15 @@ Many existing applications are able to use either the SOCKS4, SOCKS4A, or SOCKS5
 There are 2 pieces of software that work together to send SOCKS traffic through the mixnet: the `nym-socks5-client`, and the `nym-network-requester`.
 
 The `nym-socks5-client` allows you to do the following from your local machine:
-* Take a TCP data stream from a application that can send traffic via SOCKS5.
+
+* Take a TCP data stream from an application that can send traffic via SOCKS5.
```

Comment:
i dont understand what this commit is doing - just looks like you've removed then reintroduced a line?

Reply:
no, this is an added missing space, following markdown language rules

check "view file" to see the final results:

![imagen](https://github.com/nymtech/nym/assets/89636253/beb61db1-f577-41aa-9d43-477ab97b6257)